### PR TITLE
Add Login Page and Sign Up Page

### PR DESCRIPTION
**Pull request by varun13169**
Add Login Page and Sign Up Page

### CODE REVIEW

File: pages/authPage/login.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+.dui-auth-card {
+    width: fit-content;
+    min-width: 50rem;
+    height: fit-content;
+    padding: 2rem;
+    padding-left: 7.5rem;
+    padding-right: 7.5rem;
+    padding-top: 4rem;
+    padding-bottom: 4rem;
+    background-color: white;
```
Comment (itsmebipul00) [inline pages/authPage/login.css]: var could be used

**Review by itsmebipul00 — APPROVED** [review]

File: pages/authPage/login.css (hunk 1 above)
Comment (itsmebipul00) [reply pages/authPage/login.css]: good to merge with changes

**Review by dev-prabhat — CHANGES_REQUESTED** [review]
Looks good to me

**Review by dev-prabhat — APPROVED** [review]
Good to merge

File: pages/authPage/login.css (hunk 1 above)
Comment (varun13169) [reply pages/authPage/login.css]: Done, made it a variable.

**Pull request merged**